docs/sphinx_doc/en/source/tutorial/203-model.md

```diff
@@ -390,6 +393,70 @@ Here we provide example configurations for different model wrappers.
 }
 ```
 
+</details>
+<details>
+<summary>Post request API for Chat (<code><a href="https://github.com/modelscope/agentscope/blob/main/src/agentscope/models/post_model.py">agentscope.models.PostAPIModelWrapperBase</a></code>)</summary>
```

Comment:
The positioning of these APIs are unclear, and more specific descriptions are suggested: 
1. When should users try to use these wrappers? 
2. What's their requirement for the model services?